example.go

```diff
@@ -169,4 +201,23 @@ func main() {
 	if err != nil {
 		log.Fatalf("Installing package failed: %s\n", err)
 	}
+
+	marathonApp := marathon.NewDockerApplication().
+		Name("appzzz").
+		CPU(0.1).
+		Memory(64).
+		Count(2).
+		AddArgs("sleep", "36000").
+		AddEnv("MY_ENV", "my_val")
+	marathonApp.Container.Docker.Container("alpine:latest")
```

Comment:
These are just helper methods, it's possible to declare a struct directly instead.

> Note: Applications may also be defined by means of initializing a marathon.Application struct instance directly. However, go-marathon's DSL as shown above provides a more concise way to achieve the same.